src/jaegertracing/utils/UDPClient.h

```diff
@@ -59,7 +61,11 @@ class UDPClient : public agent::thrift::AgentIf {
                 << batch.spans.size();
             throw std::logic_error(oss.str());
         }
+#ifdef WIN32
+        const auto numWritten = ::send(_socket.handle(), reinterpret_cast<char*>(data), sizeof(uint8_t) * size, 0);
```

Comment:
Same question regarding casting only on Windows.

Reply:
changed to ::send for Linux and Windows.